packages/ui/src/components/Designer/Canvas/index.tsx

```diff
@@ -381,6 +381,9 @@ const Canvas = (props: Props, ref: Ref<HTMLDivElement>) => {
             onChange={(value) => {
               changeSchemas([{ key: 'data', value, schemaId: schema.id }]);
             }}
+            onCustomAttributeChange={(key, value) => {
```

Comment:
this gives flexibility to allow a ui renderer to make changes to different schema attributes, in my case `content`